tests/Middleware/ContextInterfaceMiddlewareTest.php

```diff
@@ -21,33 +21,43 @@ class ContextInterfaceMiddlewareTest extends TestCase
 {
     public function testInvoke()
     {
-        $context = new Context();
-        $context->setTag('bar', 'foo');
-        $context->mergeUserData(['foo' => 'bar']);
-        $context->mergeExtraData(['bar' => 'baz']);
+        $userContext = new Context();
+        $userContext->setData(['foo' => 'bar']);
+
+        $tagsContext = new Context();
+        $tagsContext->setData(['foo', 'bar']);
```

Comment:
This is the same as the previous one, you could mix those up in the tests; change it.

Reply:
The lines are the same, but the context instances are different and they are used in the test to ensure that the merging works for each of them